pulp_service/requirements.txt

```diff
@@ -1,5 +1,5 @@
 pulpcore==3.74.1
-pulp-rpm==3.28
+pulp-rpm==3.29.1
```

Comment:
Maybe don't also do this in this PR.

Reply:
fixed in https://github.com/pulp/pulp-service/pull/482